src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,274 @@
+package duke;
+
+import duke.exceptions.InvalidInputException;
+import duke.exceptions.InvalidTaskFormatException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.TaskEnum;
+import duke.tasks.ToDo;
+
+import data.Data;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Duke {
+    private static final String INDENT = "    ";
+    private static final String LINE = "____________________________________________________________";
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String CHAR_SPACE = " ";
+
+    // commands
+    private static final String COMMAND_EXIT = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DELETE = "delete";
+    private static final String COMMAND_SAVE = "save";
```

Comment:
Good use of constant. You might want to put these into a separate class in the future when you do more OOP increment